test/test_install/CMakeLists.txt

```diff
@@ -60,16 +54,13 @@ if(GINKGO_BUILD_HIP)
     else()
         set (GINKGO_PIC_OPTION "$<$<CONFIG:Debug>:-fPIC>")
     endif()
-    if (CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA")
-        set(TESTINSTALL_CUDA_ARCH_FLAGS "${GINKGO_CUDA_ARCH_FLAGS}")
-    endif()
     if (CMAKE_CUDA_HOST_COMPILER)
         set(TESTINSTALL_CUDA_HOST_COMPILER "-ccbin=${CMAKE_CUDA_HOST_COMPILER}")
     endif()
     hip_add_executable(test_install_hip test_install.cpp
         HIPCC_OPTIONS "-std=c++14"
         CLANG_OPTIONS "${GINKGO_PIC_OPTION}"
-        NVCC_OPTIONS "${GINKGO_CUDA_PIC_OPTION}" "${TESTINSTALL_CUDA_ARCH_FLAGS}" "${TESTINSTALL_CUDA_HOST_COMPILER}")
+        NVCC_OPTIONS "${GINKGO_CUDA_PIC_OPTION}" "${TESTINSTALL_CUDA_HOST_COMPILER}")
```

Comment:
the arch flag is missed

Reply:
The file doesn't contain any kernels, so we don't need to specify the architecture